### DISCUSSION TITLE
Project failed to download

### DISCUSSION THREAD

**Discussion by Tirex2807** [opengisch/QField#5315] [Bugs]
Hello everyone, 
I'm currently using Qfield on IOS to guide myself on the field. 
I noticed Yesterday evening after my day of field work that there was an update on the App Store, so I downloaded it. 

Since then, the app I can't open the map I just used Yesterday and it says the project failed to download. 
The error persists even if I create a new project and upload it.

Can anyone help me or have the same Issue ?

**Comment by nirvn** [comment]
@Tirex2807 , known regression with QField 3.3.0. A new version fixing this issue is rolling out as we speak.

**Reply by Tirex2807** [reply]
Any idea of when this will be out? I need Qfield to work for the field I'm doing right now :-(

**Reply by suricactus** [reply]
Just in case you didn't notice, the patch was already out by the time nirvn answered above. You can update from your store provider or download directly from https://github.com/opengisch/QField/releases/tag/v3.3.1 . Note if you ever experience similar regression, you can always downgrade to an older version that have worked for you.
